### DISCUSSION TITLE
https://harrellbm.github.io/Quartz-Garden/The-Theology-of-Dirt

### DISCUSSION THREAD

**Discussion by giscus** [harrellbm/Quartz-Garden#3] [Announcements]
# https://harrellbm.github.io/Quartz-Garden/The-Theology-of-Dirt

🌿Sapling🙂Agree Importance: 50% The Big Idea: The connection between dirt and the human person is deeply meaningful and worth investigating. Dirt as a part of Humans (used for OS📃Epistle May 2023) The Bible is full of so much rich imagery.

https://harrellbm.github.io/Quartz-Garden/The-Theology-of-Dirt

**Comment by harrellbm** [comment]
Copied from Email from Patrick while posts were down:

Made me think of this…

One day a group of scientists got together and decided that humanity had come a long way and no longer needed God.  So they picked one scientist to go and tell Him that they were done with Him.  The scientist walked up to God and said, “God, we've decided that we no longer need you.  We’re to the point where we can clone people, manipulate atoms, build molecules, fly through space, and do many other miraculous things.   So why don’t you just go away and mind your own business from now on?” God listened very patiently and kindly to the man.  After the scientist was done talking, God said, “Very well.  How about this?  Before I go, let’s say we have a human-making contest.”  To which the scientist replied, “Okay, we can handle that!” “But,” God added, “we’re going to do this just like I did back in the old days with Adam. The scientist nodded, “Sure, no problem” and bent down and picked up a handful of dirt. God wagged a finger at him and said, “Uh, uh, uh.  Put that down.  You go find your own dirt.”

And in searching for that joke to get the language right, I came upon the source of what I pasted above which I found to be a very interesting read. As I’ve said, the intersection of science and religion interest me greatly.


https://www.jpost.com/blogs/through-a-glass-darkly/gods-dirt-386925

**Reply by harrellbm** [reply]
My reply:

I love this!! Definitely makes you think about how much we rely on without even thinking about it.
